### ISSUE TITLE
InvalidCharacterError: String contains an invalid character

### ISSUE THREAD

**Issue by GuyBaele** [phylogeography/d3-renderer#14]
Depending on the tag name for the discretized branch rates when logging trees in the BEAST XML, the console log of the d3 renderer yields the following error, which results in an incorrect rendering of the lines, and in the absence of the dropdown menu's for the lines and the polygons in the browser visualization:
InvalidCharacterError: String contains an invalid character  main.js:11459
	attrConstant file:///Users/guybaele/Downloads/2/test/main.js:11459:8
	d3_selectionPrototype.each/< file:///Users/guybaele/Downloads/2/test/main.js:11758:8
	d3_selection_each file:///Users/guybaele/Downloads/2/test/main.js:11764:31
	d3_selectionPrototype.each file:///Users/guybaele/Downloads/2/test/main.js:11757:13
	d3_selectionPrototype.attr file:///Users/guybaele/Downloads/2/test/main.js:11448:13
	exports.generateLinesLayer/< file:///Users/guybaele/Downloads/2/test/main.js:22369:18
	forEach self-hosted
	exports.generateLinesLayer file:///Users/guybaele/Downloads/2/test/main.js:22358:6
	render/</< file:///Users/guybaele/Downloads/2/test/main.js:232:12
	forEach self-hosted
	render/< file:///Users/guybaele/Downloads/2/test/main.js:214:6
	d3_xhr/xhr.send/< file:///Users/guybaele/Downloads/2/test/main.js:12792:10
	event file:///Users/guybaele/Downloads/2/test/main.js:11300:41
	respond file:///Users/guybaele/Downloads/2/test/main.js:12745:10

This occurs when the XML element in BEAST looks like this:
<logTree id="treeFileLog" logEvery="5000" nexusFormat="true" fileName="2.trees" sortTranslationTable="true">
			<treeModel idref="treeModel"/>
			<trait name="rate" tag="2.rate">
				<discretizedBranchRates idref="2.branchRates"/>
			</trait>
...
</logTree>

In other words, the tag="2.rate" seems to be the cause of the problem, but I have no idea why at the moment. Rename "2.rate" to "rate" fixes the problem.
